docs/pages/components/web/sidenavigation.js

```diff
@@ -514,301 +301,133 @@ function Example() {
-            </RadioGroup>
-          }
-        >
-          {organisedBy === 'categorized' ? (
-            <React.Fragment>
-              <SideNavigation.Section label="Navigation">
-                <SideNavigation.TopItem
-                  href="https://gestalt.pinterest.systems/pageheader"
-                  label="PageHeader"
-                />
-                <SideNavigation.TopItem
-                  href="https://gestalt.pinterest.systems/tabs"
-                  label="Tabs"
-                />
-                <SideNavigation.TopItem
-                  href="https://gestalt.pinterest.systems/tooling"
-                  label="SideNavigation"
-                  badge={{ text: 'New', type: 'info' }}
-                />
-              </SideNavigation.Section>
-              <SideNavigation.Section label="Controls">
-                <SideNavigation.TopItem
-                  href="https://gestalt.pinterest.systems/radiobutton"
-                  label="RadioButton"
-                  badge={{ text: 'Deprecated', type: 'warning' }}
-                />
-                <SideNavigation.TopItem
-                  href="https://gestalt.pinterest.systems/radiogroup"
-                  label="RadioGroup"
-                />
-              </SideNavigation.Section>
-            </React.Fragment>
-          ) : (
-            <React.Fragment>
-              <SideNavigation.TopItem
-                href="https://gestalt.pinterest.systems/pageheader"
-                label="PageHeader"
+            sandpackExample={
+              <SandpackExample
+                code={headerExample}
+                name="Header example"
+                showEditor={false}
+                layout="column"
+                previewHeight={380}
               />
-              <SideNavigation.TopItem
-                href="https://gestalt.pinterest.systems/radiobutton"
-                label="RadioButton"
-                badge={{ text: 'Deprecated', type: 'warning' }}
-              />
-              <SideNavigation.TopItem
-                href="https://gestalt.pinterest.systems/radiogroup"
-                label="RadioGroup"
-              />
-              <SideNavigation.TopItem
-                href="https://gestalt.pinterest.systems/tooling"
-                label="SideNavigation"
-                badge={{ text: 'New', type: 'info' }}
-              />
-              <SideNavigation.TopItem
-                href="https://gestalt.pinterest.systems/tabs"
-                label="Tabs"
-              />
-            </React.Fragment>
-          )}
-        </SideNavigation>
-      </Box>
-    </Box>
-  );
-}
-`}
+            }
           />
         </MainSection.Subsection>
         <MainSection.Subsection
           title="Footer"
           description="Footers allow for filters, additional external links or other UI controls to be included at the bottom of the navigation."
         >
           <MainSection.Card
+            // sandpackExample={
```

Comment:
Can we remove this commented-out code?

Reply:
It's a reminder that we don't support gestalt-datepicker in Sandpack. As soon as we do so, we can use this example.